lib/simple_redlock/lockable.rb

```diff
@@ -1,8 +1,14 @@
 module SimpleRedlock
   module Lockable
-    def exclusively(key, options = {}, &block)
-      transaction do
-        redis_lock.with_lock!(resource: exclusive_key(key), **options) do
+    def exclusively(key, open_transaction: true, dont_reload: false, **options, &block)
```

Comment:
I guess it works for us the transaction default but it would be something I wouldn't expect by default, still, I think is the right change for now, in the future we can change almost everywhere in the app to avoid transactions.